upgrade_test.py

```diff
@@ -376,15 +379,6 @@ def _rollback_node(self, node, upgrade_sstables=True):
         node.run_nodetool("snapshot")
         node.stop_scylla_server(verify_down=False)
 
-        if self.params.get("enable_tablets_on_upgrade"):
```

Comment:
don't we need to remove `enable_tablets` when rolling back?

Reply:
@yarongilor please review that part

Reply:
we restore the old scylla.yaml, which has initial config, so if it was disabled in yaml, it will be restored as disabled

Reply:
one general note (perhaps it is already covered?): if no new keyspace (+table+data) is created post upgrade (with enable_tablets) , it means "tablets" are not really active or verified to run.
It might be interesting to cover such a scenario where running a mixed mode like gemini+cdc+no_tablets and a new tablets-stress.

Reply:
I don't sure, this should be done in this PR. It could be added as new test for tablets. CDC is still not supported with tablets yet.

Reply:
> one general note (perhaps it is already covered?): if no new keyspace (+table+data) is created post upgrade (with enable_tablets) , it means "tablets" are not really active or verified to run.
> It might be interesting to cover such a scenario where running a mixed mode like gemini+cdc+no_tablets and a new tablets-stress.

Great idea, is it written anywhere, so someone will actually update the upgrade tests to cover it ?

Reply:
opened https://github.com/scylladb/scylla-cluster-tests/issues/8259